tests/xtimer_usleep/main.c

```diff
@@ -25,24 +27,23 @@
 #define ONE_SEC_SLEEP     (1 * US_PER_SEC)
 #define FIVE_SEC_SLEEP    (5 * US_PER_SEC)
 #define TEN_SEC_SLEEP     (10 * US_PER_SEC)
+#define T                 (9U)
 
 int main(void)
 {
-    int t = 9;
-
-    printf("This test will print \"Slept for X sec...\" every 1, 5 and 10 seconds.\n");
-    printf("\n<======== If using pyterm, this is the time when started.\n\n");
-
-    while (t) {
+    printf("Running test %u times\n", T);
+    for (unsigned t = 0; t < T; t++) {
+        printf("now: %" PRIu32 "\n", xtimer_now_usec());
         xtimer_usleep(ONE_SEC_SLEEP);
         printf("Slept for 1 sec...\n");
+        printf("now: %" PRIu32 "\n", xtimer_now_usec());
```

Comment:
I think it would make more sense to measure the difference before and after calling xtimer_usleep either than printing 'now'. This way, even with make 'term' you can see if it looks ok or not.
Also it would remove 'printfs' calls from the timing.